resources/mail/contact.php

```diff
@@ -1,12 +1,6 @@
 <?php
 use Yiisoft\Html\Html;
 
-/* @var string $name */
 /* @var string $content */
-?>
 
-<p>
-    <?= Html::encode($content) ?>
-</p>
-
-<p><?= Html::encode($name) ?></p>
```

Comment:
Why name was removed?

Reply:
It's already picked up in the email From

Reply:
![image](https://user-images.githubusercontent.com/68641750/103160940-56868b00-47db-11eb-8066-4fe84d3d786d.png)